### ISSUE TITLE
Accessibility swarm on the Useful Links component

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2275]
**Is your feature request related to a problem? Please describe.**
Carry out a `two devs` accessibility swarm for the `UsefulLinks` component once it is completed.

**Describe the solution you'd like**
Go through the [accessibility checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
Blocked on https://github.com/bbc/psammead/issues/2222

**Comment by OlgaLyubin** [comment]
Unblocked on https://github.com/bbc/psammead/issues/2222

**Comment by DenisHdz** [comment]
❓ = Not done
✅ = Done, pass
❌ = Done, fail
👀 = Review with UX

## Checklist - Manual ##

  - ### Design 👀

    There are issues on IE to be fixed.

```
export const UsefulLinksLi = styled.li.attrs({ role: 'listitem' })`
  padding-top: ${GEL_SPACING};
  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
    display: inline-block;
    min-width: 50%;

    @supports (${grid}) {
      display:block;
      min-width: initial;
      align-self: center;
    }
    /* @supports not (${grid}) {
      display: inline-block;
      min-width: 50%;
    } */
  }
`;
```

  - ### Color contrast ✅

  - ### Color blindness ✅

  - ### High Contrast mode PC 

  - ### Color preferences changed in Firefox ✅
    
  - ### Text resized to 200%  ✅

  - ### Page zoomed to 200%  ✅

  - ### Heading order  ✅ 

  - ### Icons  ✅
   
  - ### Keyboard navigation: tab order, focus css, trace tab path ✅

```
- Given I use a keyboard only
- When I navigate to a useful links item by pressing the TAB key
- Then this must have visible state change on focus 
- And this must not be identified by colour alone 
- And be in addition to the browser default focus outline styling
```

  - ### aXe ✅

  - ### No CSS ✅

  - ### No Javascript ✅

  - ### bbc-a11y ✅


## Checklist - Assistive Technology ##

 - ###  VoiceOver OS (Mac) (Screen Reader) ✅

   > With Safari (Latest Version) on Mac OS (Latest Version)  

 - ### VoiceOver iOS (iPhone/iPad) (Screen Reader) ✅

    > Latest Version with Safari (Latest Version) on iOS (Latest Version)

 - ### TalkBack  (Screen Reader) ✅
 
   > Latest Version with Chrome (Latest Version) on Android (Latest Version)

- ### ZoomText (Screen Magnifier with Screen Reader capabilities) ✅

  > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### Dragon Naturallyspeaking  (Speech Recognition) ✅

   > Version 13 with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### JAWS  (Screen Reader) ✅

   > Version 17 with Internet Explorer 11 on Windows (XP/Vista/7/8/10) 

- ### Read&Write (Reading Solution) ✅

   > Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

- ### NVDA  (Screen Reader) ✅

   > Latest Version with Firefox (Latest Version) on Windows (XP/Vista/7/8/10)

**Comment by DenisHdz** [comment]
The IE issues have been now fixed.
